ilugd.py

```diff
@@ -6,19 +6,18 @@
 from random import choice
 import os
 
-BOTNAME = 'ILUGDbot'
-
+BOTNAME = 'YOURBOTNAMEGOESHERE'
+toggle_state_welcome = 0
 @run_async
 def send_async(bot, *args, **kwargs):
     bot.sendMessage(*args, **kwargs)
-
 logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',level=logging.DEBUG)
 
 config = configparser.ConfigParser()
 config.read('bot.ini')
 
 
-updater = Updater(os.environ['token']) # we should use env variable !!
+updater = Updater(token = '630074149:AAE5G411hgWo33tejBsHP6qxFc6eSO7S-m0') # we should use env variable !! <- I don't (^^)
```

Comment:
One must not push his token to a public repository, also this bot already has one, which is set as the environment variable, please revert the line back

Reply:
also .idea should not be pushed it should be in .gitignore